### ISSUE TITLE
css module missing 

### ISSUE THREAD

**Issue by cpugirl** [JedWatson/react-select#2290]
https://unpkg.com/react-select/dist/react-select.css
Cannot find module "/dist/react-select.css" in package react-select@2.0.0-alpha.1

**Comment by MaximAbramchuck** [comment]
+1, the same problem for me, started happening a couple of hours ago

**Comment by cristianpolo** [comment]
It seems that the file has been removed for that repo.

Previous version works: https://unpkg.com/react-select@1.2.0/dist/react-select.css.

**Comment by acidjames** [comment]
Hi,

how do you change from one to another ? I'm using 1.2.0 and i still have 2.0 etc in my code 

tried clearing yarn cache to no avail :(

**Comment by robhinds** [comment]
Have the same problem - looks like the file is still in the build on npm, so assume it was some problem publishing on to unpkg.

**Comment by pedall** [comment]
also missing for me

**Comment by crilopesp** [comment]
I just use it in in my index.html.

`<link rel="stylesheet" href="https://unpkg.com/react-select@1.2.0/dist/react-select.css?v=2">`

You can use it as it is in Readme: 

`import 'react-select/dist/react-select.css';`

**Comment by ryanmartinneutrino** [comment]
Hopefully the push to unpkg.com is fixed for the css file, the other files are there....

**Comment by 416serg** [comment]
Still an issue for me 😢

**Comment by kevinzwhuang** [comment]
It's because `2.0.0-alpha.1` was released with the `latest` tag on npm - replacing `v1.2.0` as the `latest` stable version of react-select on npm.

`2.0.0-alpha.1` should have been released on a different tag than __latest__, preferably __next__ or some other tag that denotes experimental releases (https://docs.npmjs.com/cli/dist-tag#purpose)

@JedWatson I'd suggest unpublishing `react-select@2.0.0-alpha.1` to restore react-select for end-users, and republishing it under the `next` tag. 

```
npm unpublish
npm publish --tag next
```

This way, we restore react-select to those depending on it from unpkg from v1, while retain access to the experimental build for those who want to, through `npm install react-select@2.0.0-alpha.1` or `npm install react-select@next`.

**Comment by JedWatson** [comment]
Sorry, I totally missed that implication of publishing the alpha version of 2.0 to npm. Will try and sort this out asap. In the meantime, you should be able to fix this by ensuring that:

## If you're using npm with a bundler (parcel, webpack, rollup, etc)

Ensure your dependency in package.json is:

```
"react-select": "^1.2.0"
```

## If you're using the umd build from unpkg:

Ensure your script tag points at the following src:

```
<script src="https://unpkg.com/react-select@1.2.0/dist/react-select.js"></script>
```

If you're installing react-select in your project for the first time:

```
yarn add react-select@1.2.0
```

or

```
npm install --save react-select@1.2.0
```

**Comment by JedWatson** [comment]
I believe this has been resolved now by updating the tags for `react-select`

They are now:

```
latest: 1.2.0
next: 2.0.0-alpha.1
```

The unpkg url `https://unpkg.com/react-select` now correctly resolves to `https://unpkg.com/react-select@1.2.0` and the npm page https://www.npmjs.com/package/react-select is showing `v1.2.0` as the latest version (with the correct readme)

**Comment by lausellduane** [comment]
@JedWatson I got the same error today with the latest version react-select@2.0.0 that was published a month ago. What can I do to fix this?

**Comment by SeanK1191** [comment]
Just seeing this today, file is missing in node_modules on version 2.0.0

**Comment by kwierchris** [comment]
Same problem here. Trying to switch to version 2.0.0 in order to use the optgroup featrue but the CSS file seems not to exist anymore and without at least the multi select feature is broken.

**Comment by maggi578** [comment]
Facing the same issue. CSS file seems to be missing.

**Comment by cskru** [comment]
+1

**Comment by NicoleRauch** [comment]
The CSS is still not available for react-select 2.2.0. Any chance of fixing this?

**Comment by damoncasale** [comment]
I guess this never got answered????  Having the same issue, a whole major version later.  -_-

**Comment by adi-endor** [comment]
same problem

**Comment by mattvb91** [comment]
any chance anyone has the full css? I have missing styles on nextjs and need the full css sheet i can add globally

**Comment by mthines** [comment]
https://github.com/JedWatson/react-select/issues/3124#issuecomment-1021107495

Even though this REALLY isn't an ideal solution, it fixes the problem temporarily until the styles works when using SSR (Nextjs, etc)

I've copied most of the styles not being applied to a `*.module.scss` file which you can import and use.

#### This is a fix for people using `SCSS Modules` for [React-Select](https://react-select.com/home)

1. Download and import the [`react-select_overrides.module.scss`](https://gist.githubusercontent.com/mthines/fa2a1b8bb5058b9ca2e793934ab1918b/raw/66ac0f97c588bd6d984e188d85cb878203b91188/react-select_overrides.module.scss) file and put it next to your `styles.module.scss` file.

Like so.

### Your styles file
```scss
@use './react-select_overrides.module.scss' as *;

.reactSelect {
  // your custom styling
}
```

2. Add the SCSS Module classNames to the project
```tsx
return (
  <ReactSelect
    className={styles.reactSelect}
    classNamePrefix="react-select"
  />
)
```

https://gist.github.com/mthines/fa2a1b8bb5058b9ca2e793934ab1918b

**Comment by SafiaNuzhath** [comment]
App is not recognizing css in PROD env and styling is breaking. Local env works fine 

Any fix for this issue for "react-select": "^5.7.0" ??

**Comment by honia19** [comment]
> App is not recognizing css in PROD env and styling is breaking. Local env works fine
> 
> Any fix for this issue for "react-select": "^5.7.0" ??

the same thing. Locally is okay, after deploying the styles weren't applied. I use ruby on rails monolith
